api/nodes/serializers.py

```diff
@@ -107,6 +108,11 @@ class NodeSerializer(JSONAPISerializer):
         related_view_kwargs={'node_id': '<parent_id>'}
     )
 
+    ultimate_parent = RelationshipField(
+        related_view='nodes:node-detail',
+        related_view_kwargs={'node_id': '<ultimate_parent>'}
```

Comment:
This will need to change to get the `._id` from the newly renamed `root`. L41 above will also need to be renamed.